### ISSUE TITLE
ER: left indentation last row of icons

### ISSUE THREAD

**Issue by LucaVarriale** [doogie2301/pinup-popper-browser#7]
The last row of icons is centered and looks inconsistent with the rest of the page.

![image](https://user-images.githubusercontent.com/62959264/87217071-747a3500-c2fa-11ea-9157-745f5a1b729a.png)

**Comment by doogie2301** [comment]
yeah I didn't like this either, but I couldn't find an easy fix when I first looked at it.  i'll give it another try when i get a chance.

**Comment by D3Troy** [comment]
There seems to be an easy fix for this.

If you replace m-auto with my-auto within `views/index.pug` that applies auto margins only on the vertical axis (keeping vertical centering) and removes horizontal auto margins, so items will left-align naturally when a row isn't full.

@doogie2301: I have created the PR https://github.com/doogie2301/pinup-popper-browser/pull/82 to include this in your project.
